plugins/Live/Reports/GetLastVisitsDetailsWidget.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Copyright (C) Piwik PRO - All rights reserved.
+ *
+ * Using this code requires that you first get a license from Piwik PRO.
+ * Unauthorized copying of this file, via any medium is strictly prohibited.
+ *
+ * @link http://piwik.pro
+ */
+
+namespace Piwik\Plugins\Live\Reports;
+
+use Piwik\Plugins\Live\Visualizations\VisitorLogWidget;
+
+class GetLastVisitsDetailsWidget extends Base
```

Comment:
When I tried it, it showed my the real time widget and I couldn't notice a difference so I got confused as there's already a real time widget and a visitor log widget. Still not 100% sure what the difference is for that new one sorry.